src/main/scala/components/memory/memory-noncombin-ports.scala

```diff
@@ -30,6 +41,17 @@ class INonCombinMemPort extends ICombinMemPort {
  * The I/O for this module is defined in [[DMemPortIO]].
  */
 class DNonCombinMemPort extends BaseDMemPort {
+  // To fake combinational writes
+  val dmemBusy  = Reg (new Bool)
```

Comment:
Why not RegInit like above?

Reply:
It was a forgotten change from when I was trying to fix buggy behavior with these particular registers. It relates to another comment you had about the register being connected to DontCare.